integration/core/test_cli.py

```diff
@@ -657,6 +657,7 @@ def restore_backup(engine_manager_client,  # NOQA
         rs = json.loads(subprocess.check_output(status_cmd).strip())
         for status in rs.values():
             assert 'state' in status.keys()
+            assert status['backupURL'] != ""
```

Comment:
And move `time.sleep(3)` to the bottom of the loop, and update to etc. `time.sleep(RETRY_INTERVAL)`